contracts/contracts/ramps/paylah/PaylahRamp.sol

```diff
@@ -486,12 +492,24 @@ contract PaylahRamp is Ownable {
 
     /* ============ Governance Functions ============ */
 
+    /**
+     * @notice GOVERNANCE ONLY: Adds a payment link to the paymentLinks mapping. This payment link has been verified to be
+     * tied to a specific off-ramp id hash hash(alias + 4 mobile digits). All confirmation e-mails sent to on-rampers that
+     * paid the payment link above will contain information to build the matching off-ramp id hash.
+     *
+     * @param _offRampIdHash   Off-ramp id hash that the payment link is tied to calculated as hash(alias + 4 mobile digits)
+     * @param _paymentLink     Payment link that has been verified to be tied to the off-ramp id hash
+     */
+    function addVerifiedPaymentLink(bytes32 _offRampIdHash, bytes calldata _paymentLink) external onlyOwner {
```

Comment:
lgtm. We can either tie it to offRampIDHash (hash of alias + 4-digit mobile number) or tie it to the raw alias + raw 4-digit mobile number. But the tie looks good to me. Also instead of `onlyOwner` we could make it be a whitelisted pair of addresses.

Reply:
Realizing this is also something we could do off-chain. Need not be a hard requirement on-chain.

Reply:
can you expand on the whitelisted pair of addresses idea? And what do you mean by need not be a hard requirement off-chain.

A better flow we could consider is this:
1. User submits payment link to our server
2. We make a small payment and extract the necessary information
3. We sign the resulting `offRampIdHash` and paymentLink with a key of ours
4. User submits the signature, `offRampIdHash`, and paymentLink to the contract and we verify the signature matches
5. `offRampIdHash` and paymentLink get set for the deposit

cc @asoong @richardliang for thoughts on this as well